<launch>
	<node pkg="behavior_logic" name="fake_gps_data" type="fake_gps_data.py" output="screen"></node>
	<!-- <node pkg="behavior_logic" name="fake_cmd_msg" type="fake_cmd_msg.py" output="screen"></node> -->
	<node pkg="behavior_logic" name="behavior_logic_node" type="behavior_logic_node.py" output="screen"></node>
	<!-- <node pkg="behavior_logic" name="behavior_logic_test" type="service_test.py" output="screen"></node> -->
	<include file="$(find rosbridge_server)/launch/rosbridge_websocket.launch" > 
     <arg name="port" value="9090"/>
  	</include>
	<!-- <node pkg="behavior_logic" name="rpi_action_server" type="rpi_action_server.py" output="screen"></node> -->
</launch>